packages/core/e2e/Customer.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { CustomerStepPreset, expect, test } from '@bigcommerce/checkout/test-framework';
+
+test.describe('Customer', () => {
```

Comment:
I just have one comment if we should add this test here or in functional tests repo?

Reply:
As discussed offline, we will create new e2e tests in the functional tests repository.

For the time being, we can use `// TODO: CHECKOUT-9010` to mark any playwright test work and ignore the code coverage with `/* istanbul ignore next */`.